R/zchunk_batch_all_aglu_emissions_IRR.xml.R

```diff
@@ -1,14 +1,14 @@
-#' module_emissions_batch_all_aglu_emissions_IRR.xml
```

Comment:
Ah!  `zchunk_batch_all_aglu_emissions_IRR.xml.R` generates the xml from L2111. I didn't mean to change it. I went back and found the original version of the file and can include it in my next commit.